src/App.jsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import logo from './logo.svg';
 import './App.css';
+import Home from './pages/HomePage';
+import ScholarshipListPage from './pages/ScholarshipListPage';
+import ScholarshipDetailsPage from './pages/ScholarshipDetailsPage';
 
 function App() {
   return (
+    <Router>
+      <Switch>
+        <Route path="/scholarships/:id" component={ScholarshipDetailsPage} />
+        <Route path="/scholarships" component={ScholarshipListPage} />
+        <Route path="/" component={Home} />
```

Comment:
Reorder this. Look at the Switch [here](https://reactrouter.com/web/guides/quick-start)

Reply:
Nvm, it looks like it was fixed.